Makefile

```diff
@@ -34,11 +34,11 @@ SRCEXT 		:= \([$(CEXT)$(AEXT)$(SEXT)]\|$(CPPEXT)\)
 #SRCS = $(wildcard *.S)
 #PROGS = $(patsubst %.S,%,$(SRCS))
 BASEDIR = ${WALLY}/addins/cvw-arch-verif
-SRCDIR64 = ${BASEDIR}/tests/rv64/I
-SRCDIR32 = ${BASEDIR}/tests/rv32/I
+SRCDIR64 = ${BASEDIR}/tests/rv64
+SRCDIR32 = ${BASEDIR}/tests/rv32
+WALLYEXTS = 	$(shell find ${SRCDIR64} ${SRCDIR32} -type d | sort)
```

Comment:
This variable is no longer doing anything

Reply:
@jordancarlin I left the variable there for extensibility, in case the folder structure needs to be changed. It also makes it more readable

Reply:
Sounds good. Just double checking.